### ISSUE TITLE
Rhagonycha limbata species is missing

### ISSUE THREAD

**Issue by kazlauskis** [NERC-CEH/irecord-app#271]
From the website [forum](https://irecord.org.uk/forum/mobile-applications/rhagonycha-limbata-not-searchable). 

Searching for _Rhagonycha limbata_ or _Rhagonycha nigriventris_ shows no results in the app. We are using this report to fetch the species dictionary into the app: `reports/projects/irecord/taxa/taxa_list_for_app.xml?taxon_list_id=15`. Someone at the website-end should look into why this report doesn't return the species.

This is what I would receive for "_Rhagonycha limbata_" If I searched it on the iRecord website:

```
{
    "taxa_taxon_list_id": "167032",
    "searchterm": "Rhagonycha limbata Thomson, 1864",
    "highlighted": "'''<b>Rhagonycha<\/b> <b>limbata<\/b>'''",
    "taxon": "Rhagonycha limbata",
    "authority": "Thomson, 1864",
    "language_iso": "lat",
    "preferred_taxon": "Rhagonycha nigriventris",
    "preferred_authority": "Motschulsky, 1860",
    "default_common_name": null,
    "taxon_group": "insect - beetle (Coleoptera)",
    "preferred": "f",
    "preferred_taxa_taxon_list_id": "574647",
    "taxon_meaning_id": "71060",
    "external_key": "NHMSYS0021366879",
    "organism_key": "NBNORG0000089341",
    "taxon_group_id": "101",
    "parent_id": "188188",
    "identification_difficulty": null,
    "id_diff_verification_rule_id": null,
    "taxon_rank_sort_order": "300",
    "taxon_rank": "Species"
}
```

**Comment by sacrevert** [comment]
This still seems to be an issue @kitenetter

**Comment by sacrevert** [comment]
Given that the UKSI update in the warehouse is now complete (https://github.com/BiologicalRecordsCentre/iRecord/issues/1880), can we refresh the app's taxon dictionary? @kitenetter @kazlauskis

**Comment by kazlauskis** [comment]
@sacrevert I've pulled in the latest dictionary, but the species is still missing from the report.

**Comment by sacrevert** [comment]
I've reinstalled my app and can see no changes (e.g. Ervilia hirsuta is absent, when it should be the current accepted name for Vicia hirsuta as on the website)

**Comment by johnvanbreda** [comment]
That should be fixed now - there were some updates required in the cache_taxon_searchterms table after all the post UKSI-update data fixes which hadn't been applied.

**Comment by sacrevert** [comment]
@kazlauskis Can you let us know when to update the app to test? Thanks!

**Comment by kazlauskis** [comment]
@johnvanbreda I can see `Rhagonycha nigriventris` and ` Ervilia hirsuta` in the report, but no `Rhagonycha limbata`.

**Comment by sacrevert** [comment]
That's correct I think, because `Rhagonycha limbata` is a junior synonym of `R. nigriventris` and IIRC the app does not currently present synonyms

**Comment by kazlauskis** [comment]
I have pulled in the latest species list now. @sacrevert, can you check if the [demo app](https://irecord.app.flumens.io/demo.html) has the correct species now?

This will come out in the next app release (TBD).

**Comment by sacrevert** [comment]
>@sacrevert, can you check if the [demo app](https://irecord.app.flumens.io/demo.html) has the correct species now?

Looks good to me
